<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="Assignment1_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Assignment 1</string>
    </property>
    <widget class="QGroupBox" name="SystemSimulation_box">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>631</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>System Simulation</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="NoiseSimulation_box">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>631</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Noise Simulation</string>
     </property>
    </widget>
    <widget class="QSlider" name="mu_noise">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sigma_noise">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="LinearBottom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ExponentialBottom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Exponential</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="LogarithmicBottom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Logarithmic</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollBar" name="a">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="b">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="lambda_exp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="mu_exp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="a_log">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="b_log">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="RunSimulationBottom">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>490</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Simulation</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="GaussianNoise">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gussian</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="UniformNoise">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Uniform</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="a_noise">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="b_noise">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>410</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
